metals/src/main/scala/scala/meta/internal/metals/ScalaTarget.scala

```diff
@@ -60,7 +66,7 @@ case class ScalaTarget(
 
   def baseDirectory: String = info.baseDirectory
 
-  def options: List[String] = scalac.getOptions().asScala.toList
+  lazy val options: List[String] = scalac.getOptions().asScala.toList
```

Comment:
This is used twice at least [there](https://github.com/scalameta/metals/blob/main/metals/src/main/scala/scala/meta/internal/metals/BuildTargetInfo.scala#L85)